colour/temperature/tests/test_ohno2013.py

```diff
@@ -122,21 +112,31 @@ def test_uv_to_CCT_Ohno2013(self):
         definition.
         """
 
+        CCT = np.linspace(1_000, 100_000, 3_000)
+        duv = np.linspace(-0.01, 0.01, 10)
```

Comment:
**Minor**

We should probably use 0.05 here which is the limit at which the concept of CCT is not valid anymore.

Reply:
Sadly, Ohno’s method is actually only accurate to +/- 1K for values less than 0.003 when the table has a spacing of 1.01. I had to improve the spacing and write my own correction function to extend the accuracy even to this level. 

It is, unfortunately(?) only an approximation. And of course, a more dense table is also slower.